frisc/test/cpu.txt

```diff
@@ -0,0 +1,23 @@
+JSON files don't like when you put comments there, so the test cases are explained here. 
+The start of a new test case is indicated by the reset signal going high.
+The way tests are verified now is that the output of the ALU is wired to the output of the CPU, so we can inspect that.
```

Comment:
Argh, that's annoying. Perhaps we can change our JSON parsing logic to support comments using [another JSON library](https://stackoverflow.com/questions/29959191/how-to-parse-json-file-with-c-style-comments)